app/src/main/java/org/sdc/ttu/sdc_test_game_app/Login/LoginPresenter.java

```diff
@@ -4,12 +4,12 @@
 import android.support.annotation.NonNull;
 
 
-public class LoginPresenter implements LoginInterface.Presenter {
+class LoginPresenter implements LoginInterface.Presenter {
```

Comment:
@asclines it kept telling me I could make it package-private because it is only used int the Login package